internal/sqlitestore/team.go

```diff
@@ -0,0 +1,30 @@
+package sqlitestore
+
+import (
+	"github.com/jinzhu/gorm"
+	"github.com/writhe/kudosaurus"
+)
+
+// Team - person model
+type Team struct {
```

Comment:
I don't see why this type has to be exported. This is some internal model of sqlite storage. I guess it could be made internal. Same goes for other model types.